content/en/docs/reference/score-spec-reference.md

```diff
@@ -142,7 +123,7 @@ resources:
 
 It is up to the Score implementation (CLI) on how and when the resource reference is resolved, and when the referenced values' substitution occurs.
 
-For example, `score-compose` would convert resource properties into environment variables references in a resulting `compose.yaml` configuration file, and produce a reference `.env` file that the user can then populate. For more information, see the [.env file](https://docs.docker.com/compose/environment-variables/#the-env-file).
+For example, `score-compose` would convert resources into environment variables references in a resulting `compose.yaml` configuration file, and produce a reference `.env` file that the user can then populate. For more information, see the [.env file](https://docs.docker.com/compose/environment-variables/#the-env-file).
```

Comment:
I'm not sure if this is correct, specifically referring to "`score-compose` would convert resources into environment variables" - @ghen what do you think? Would be great to discuss a more accurate wording